### DISCUSSION TITLE
Chrome: no reaction when clicking to download backup file

### DISCUSSION THREAD

**Discussion by andruccho** [Authenticator-Extension/Authenticator#624] [General]
Button just do nothing.
Latest Chrome.
Storage location is set as local.
Maybe need any extra tricks in Chrome settings?

**Comment by andruccho** [comment]
I only see that chrome seems trying to download something, buh no new files in downloads
Also tried password protected backup -- it's the same
<img width="59" alt="Screenshot 2021-02-13 at 01 57 43" src="https://user-images.githubusercontent.com/23392714/107831435-1e450700-6d9f-11eb-943b-e7023ad5d48c.png">

**Comment by mymindstorm** [comment]
If you hit Ctrl+j do you see the file?
